### DISCUSSION TITLE
reference/docs/cpp

### DISCUSSION THREAD

**Discussion by giscus** [jaywcjlove/reference#444] [Q&A]
# reference/docs/cpp

https://wangchujiang.com/reference/docs/cpp.html

**Comment by Chenwkhalil** [comment]
！！！

**Comment by HyperionCHI** [comment]
C++数值部分的`std::array<int, 3> = {92, 97, 98};`这一行，是否少了数组名称，是否应修改为`std::array<int, 3> marks = {92, 97, 98};`?

**Reply by jaywcjlove** [reply]
已经修改了 @ChaunceyCHI

**Comment by HyperionCHI** [comment]
逻辑运算符中缺少"或"运算符
